File: src/engsoft/jogo/patos/Nao_Pula.java
package engsoft.jogo.patos;

public class Nao_Pula implements Padrao_Pular {

	
	public Nao_Pula()
	{
			
	}
	
	public String pular() {
		return "Esse pato nao Pula. Altura: " + getAltura();
	}

	public double getAltura() {
		return 0;
	}

}

File: Nao_Pula.ts
import { Padrao_Pular } from "./Padrao_Pular";

export class Nao_Pula implements Padrao_Pular {
    pular(): string {
        return "Este pato não pula.";
    }

    getAltura(): number {
        return 0;
    }
}
